### ISSUE TITLE
Can't run `cargo fullstack` using poem server

### ISSUE THREAD

**Issue by lynn** [Wulf/create-rust-app#64]
I ran `create-rust-app myapp` and selected sqlite and poem. Setup went smoothly, but when running `cargo fullstack` I ran into some compiler errors in `poem_utils.rs`:

```
   Compiling tera v1.17.1
   Compiling poem-derive v1.3.48
   Compiling lettre_email v0.9.4
   Compiling diesel_migrations v2.0.0
   Compiling poem v1.3.48
   Compiling create-rust-app v8.0.2
error[E0432]: unresolved import `http`
 --> C:\Users\lynn\.cargo\registry\src\github.com-1ecc6299db9ec823\create-rust-app-8.0.2\src\util\poem_utils.rs:1:5
  |
1 | use http::{StatusCode, Uri};
  |     ^^^^ help: a similar path exists: `poem::http`

error[E0433]: failed to resolve: use of undeclared crate or module `tokio`
   --> C:\Users\lynn\.cargo\registry\src\github.com-1ecc6299db9ec823\create-rust-app-8.0.2\src\util\poem_utils.rs:119:16
    |
119 |     let file = tokio::fs::read(path).await.unwrap();
    |                ^^^^^ use of undeclared crate or module `tokio`

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `create-rust-app` due to 2 previous errors
```

I'm using create-rust-app v8.0.2 and poem v1.3.48, and I'm running Windows 10.

```
> cargo --version
cargo 1.64.0 (387270bc7 2022-09-16)
> rustc --version
rustc 1.64.0 (a55dd71d5 2022-09-19)
```

**Comment by lynn** [comment]
Aha, I needed to edit `myapp/Cargo.toml` and enable features of create-rust-app:

```diff
 [dependencies]
 futures-util = "0.3.21"
-create-rust-app = {version="8.0.2", default-features = false, features=["plugin_dev", "plugin_auth", "plugin_container", "database_sqlite", "backend_poem"]}
+create-rust-app = {version="8.0.2", default-features = false, features=["plugin_dev", "plugin_auth", "plugin_container", "database_sqlite", "backend_poem", "http", "tokio"]}
 serde_json = "1.0.79"
 tracing-subscriber = "0.3.7"
 tsync = "1.2.1"
```

Maybe these features should be enabled automatically, at least when selecting "poem"?

**Comment by Wulf** [comment]
Thanks for catching this @lynn, it should be fixed in `8.1.0`

**Comment by Wulf** [comment]
Closing for now, feel free to reopen :)
